packages/calcite-components/src/components/list/usage/DragEnabledNested.md

```diff
@@ -1,19 +1,24 @@
 ```html
-<calcite-list drag-enabled group="mygroup" selection-mode="multiple">
-  <calcite-list-item open label="My Item 1" description="hello world">
-    <calcite-list drag-enabled group="mygroup" selection-mode="multiple">
-      <calcite-list-item open label="My Item 2" description="hello world">
-        <calcite-list drag-enabled group="mygroup" selection-mode="multiple">
-          <calcite-list-item label="My Item 3" description="hello world">
-            <calcite-list drag-enabled group="mygroup" selection-mode="multiple"></calcite-list>
-          </calcite-list-item>
-          <calcite-list-item label="My Item 4" description="hello world"></calcite-list-item>
-        </calcite-list>
-      </calcite-list-item>
-      <calcite-list-item label="My Item 5" description="hello world"></calcite-list-item>
-    </calcite-list>
+<calcite-list drag-enabled group="Birds">
```

Comment:
To also showcase the `selectionMode`, WDYT about adding it into this usage example? Or creating a separate one to showcase the functionality?

Reply:
Adding a separate example for selectionMode.